pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -41,11 +41,18 @@ def __init__(self, trainer: "pl.Trainer", resume_from_checkpoint: Optional[_PATH
         self._loaded_checkpoint: Dict[str, Any] = {}
 
     @property
-    def hpc_resume_path(self) -> Optional[str]:
+    def auto_restart_path(self) -> Optional[str]:
         dir_path_hpc = str(self.trainer.weights_save_path)
         max_version = self.max_ckpt_version_in_folder(dir_path_hpc, "hpc_ckpt_")
         if max_version is not None:
             return os.path.join(dir_path_hpc, f"hpc_ckpt_{max_version}.ckpt")
+        if (
+            dir_path_hpc is not None
+            and os.path.isdir(dir_path_hpc)
+            and ".pl_auto_save.ckpt" in os.listdir(dir_path_hpc)
```

Comment:
side note: we should consider making this a class attribute and perhaps removing the leading period.

Reply:
Would do, but in another PR.